guides/client/bindings.md

```diff
@@ -346,7 +346,9 @@ can be used. For example, to animate an element on mount:
 ```
 
 If `phx-mounted` is used on the initial page render, it will be invoked only
-after the initial WebSocket connection is established.
+after the initial WebSocket connection is established. An exception to this is
+elements that are defined outside of a LiveView. For those, `phx-mounted` will
+run immediately when `liveSocket.connect()` is called.
```

Comment:
Perhaps I would phrase this that `phx-mounted` is ran at the earliest opportunity once the page is loaded. For content outside of a LiveView, that's as soon as `liveSocket.connect()`. For other content, it is as soon as it is received from the server and mounted into the page.